### ISSUE TITLE
need update tsconfig.json

### ISSUE THREAD

**Issue by pomazanbohdan** [benawad/node-ts-graphql-boilerplate#1]
"allowSyntheticDefaultImports": false,
     "emitDecoratorMetadata": true,
     "esModuleInterop": true,
     "experimentalDecorators": true,
     "noFallthroughCasesInSwitch": true,
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "removeComments": true,
     "strict": true,
     "strictFunctionTypes": true,
     "strictNullChecks": true,
     "strictPropertyInitialization": false

**Comment by benawad** [comment]
yes, but I'm going to leave it as is so it matches the video. But I will keep this issue open for visibility
